composer.json

```diff
@@ -4,7 +4,7 @@
   "keywords": ["laravel", "preset", "tailwindcss"],
   "license": "MIT",
   "require": {
-    "laravel/framework": "^7.0",
+    "laravel/framework": "^8.0",
```

Comment:
```suggestion
    "laravel/framework": "^7.0|^8.0",
```

Reply:
Shouldn't it support both?

Reply:
Yes, good point. When I did this earlier I clearly didn't have enough coffee in me!